lib/matplotlib/tests/test_colors.py

```diff
@@ -156,6 +156,193 @@ def test_LogNorm():
     assert_array_equal(ln([1, 6]), [0, 1.0])
 
 
+def test_FuncNorm():
```

Comment:
These should probably either be broken into separate tests or parameterized cause if one of the tests fails,  it would likely require going back to the source to figure which subcategories failed.